README.md

```diff
@@ -128,6 +128,8 @@ Then, whenever the framework content is changed:
 - alternatively, run `npm run frontend-build:watch` to automatically rebuild the framework content whenever a framework YML file changes
 - generate schemas into the API's `json_schema` directory using the `scripts/generate-schemas.py` script in this repo.
 
+Alternatively, if you think `bower link` is more trouble than it's worth, you can just symlink the your project's `bower_components/digitalmarketplace-frameworks` to wherever this repo lives on your local filesystem and it's 99% as good as the above.
```

Comment:
It requires global (home-directory-wide) configuration in a nasty way and its complexity hides the fact that it's not *really* any more robust than `ln -s`. `ln -s` doesn't hide what it does behind a wall of 3 or 4 bad design decisions.